apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -133,7 +133,10 @@
     public static final OfficialComponent PLAY = new OfficialComponent(68, "Play");
 
     public static final OfficialComponent CASSANDRA_JAVA_DRIVER = new OfficialComponent(69, "cassandra-java-driver");
-
+  
     public static final OfficialComponent LIGHT_4J = new OfficialComponent(71, "Light4J");
 
+    public static final OfficialComponent PULSAR_PRODUCER = new OfficialComponent(73, "pulsar-producer");
```

Comment:
Is No.72 reserved for others?

Reply:
Yes. Definition of pulsar server.

Reply:
I was confused at first, seems missing one of them 😄